app/client/src/store/mutations.ts

```diff
@@ -0,0 +1,5 @@
+export default {
+  setVersions(state:any, versioninfo:object) {
```

Comment:
If this `any` is expected, could you exclude linting this line so these automatic comments below don't get distracting? If this is getting changed in an upcoming iteration don't worry

Reply:
I think I will later add a type for this like here: https://github.com/mrc-ide/comet/blob/main/src/app/static/src/store/state.ts, but thought I'd do that once I have figured out how the state will actually look like.

Reply:
Have disabled the linting in that line for now